web/src/features/charts/bar-breakdown/constants.ts

```diff
@@ -7,3 +9,18 @@ export const RECT_OPACITY = 0.8;
 export const X_AXIS_HEIGHT = 15;
 export const DEFAULT_FLAG_SIZE = 16;
 export const SCALE_TICKS = 4;
+
+export const iconHeight: { [mode in ElectricityModeType]: string } = {
+  solar: '10',
+  wind: '8',
+  hydro: '7',
+  'hydro storage': '8',
+  'battery storage': '6',
+  biomass: '9',
+  geothermal: '8',
+  nuclear: '8',
+  gas: '9',
+  coal: '9',
+  oil: '9',
+  unknown: '8',
+};
```

Comment:
Wouldn't it make more sense to get @Alportan to make SVGs in the same height so we don't need all the code around the different heights? :)

Reply:
I don't even think they need to be the same size as you can override that in the SVG directly with the height and width parameters.

Reply:
I'll create an SVG with the same proportions (8X8 px) and play with the negative space inside the frame itself.